/oak:
  ros__parameters:
    driver:
      i_enable_ir: true
    pipeline_gen:
      i_nn_type: none
      i_pipeline_type: rgbd
      i_enable_vio: true
    vio:
      i_fps: 30.0
    stereo:
      i_synced: true
      i_depth_preset: DEFAULT
    rgb:
      i_synced: true
    imu:
      i_batch_report_threshold: 1
      i_max_batch_reports: 10
      i_enable_rotation: false
